geth/common/types.go

```diff
@@ -171,9 +173,21 @@ type TxQueueManager interface {
 	DiscardTransactions(ids string) map[string]RawDiscardTransactionResult
 }
 
+// JailExecutor defines an interface which exposes method to be executed
+// against a Jail vm.
+type JailExecutor interface {
+	EvalExec(string) error
+	Run(string) (otto.Value, error)
+	Exec(string) (otto.Value, error)
```

Comment:
As I can see, Run runs the code on the underlying `*otto.Otto` without loopping. But now that we utilize `*loop.Loop`, shouldn't we run everything on it instead? I'd myself remove this method. Please, correct me if I'm wrong or misunderstand something.